client/src/pages/registry/mentor.tsx

```diff
@@ -243,20 +243,13 @@ function Page(props: Props) {
             <Row gutter={defaultRowGutter}>
               <Col {...defaultColumnSizes}>
                 <Form.Item
+                  help="We need your location for understanding audience and use it for mentor distribution. If you live close to any city from the list, please choose it."
```

Comment:
text need to be adapted for mentors registry page. e.g.
`We need your location for students distribution.`

Also `to any city from the list` this line doesn't make any sense anymore

Reply:
done